docs/guides/create_service_existing_agent.md

```diff
@@ -1 +1,212 @@
+
+3. **Test the service using a local deployment.** This is the recommended approach in order to test your agent service before you publish it to the IPFS. This process should be familiar to you if you have followed the [quick start](./quick_start.md) guide.
+
+    1. Prepare a JSON file `keys.json` containing the addresses and keys of the four agents that make up the agent service. Below you have some sample keys for testing:
+
+        !!! warning "Important"
+            Use these keys for testing purposes only. **Never use these keys in a production environment or for personal use.**
+
+            ```json
+            [
+              {
+                  "address": "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
+                  "private_key": "0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a"
+              },
+              {
+                  "address": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
+                  "private_key": "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
+              },
+              {
+                  "address": "0x976EA74026E726554dB657fA54763abd0C3a0aa9",
+                  "private_key": "0x92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e"
+              },
+              {
+                  "address": "0x14dC79964da2C08b23698B3D3cc7Ca32193d9955",
+                  "private_key": "0x4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356"
+              }
+            ]
+            ```
+
+
+    2. Build the Docker image of the service agents:
+        ```bash
+        autonomy build-image
+        ```
+        After the command finishes building it, you can see that it has created the image by executing:
+        ```bash
+        docker image ls | grep hello_world
+        ```
+
+        Note that if you have already executed the steps in the [quick sart](./quick_start.md), you should already have an image of the `hello_world` agent.
+
+
+    3. Build the deployment setup for the service:
+        ```bash
+        autonomy deploy build keys.json
+        ```
+
+    4. The build configuration will be located in `./abci_build`. Run the deployment locally using
+        ```bash
+        cd abci_build
+        autonomy deploy run
+        ```
+        You can cancel the local execution by pressing `Ctrl-C`, and return to the service parent folder `cd ..`.
+
+
+
+4. **Publish the service on the [IPFS](https://ipfs.io/).** This will make the service available for other developers to fetch it.
+
+    !!! warning "Important"
+        Before publishing any service or agent, make sure that all the files and folders no longer needed are deleted. In this example,
+            ```bash
+            rm keys.json
+            rm -rf abci_build
+            ```
```

Comment:
We need to make sure publish command takes care of this. https://github.com/valory-xyz/open-autonomy/issues/1315